from shared.basic_exception import CustomException


class FileNotFoundException(CustomException):
    pass
